cpp/src/io/utilities/datasource.cpp

```diff
@@ -391,6 +391,7 @@ class user_datasource_wrapper : public datasource {
   datasource* const source;  ///< A non-owning pointer to the user-implemented datasource
 };
 
+#ifdef CUDF_KVIKIO_REMOTE_IO
```

Comment:
Please also wrap the corresponding header by this macro too.

Reply:
done